scripts/inference/ft/README.md

```diff
@@ -0,0 +1,25 @@
+# MPT with FasterTransformer (FT)
+This folder contains a script that converts MPT checkpoints to the
+[FasterTransformer](https://github.com/NVIDIA/FasterTransformer) format. This makes the checkpoints compatible with the
+FasterTransformer library, which can be used to run transformer models on GPUs.
+
+You can either pre-download the model in a local dir or directly provide the HF hub model name to convert the MPT
+checkpoint to FasterTransformer format.
+
+## Pre-Download the Model and Convert
+```
+apt update
+apt install git-lfs
+git lfs install
+git clone https://huggingface.co/mosaicml/mpt-7b
+# This will convert the MPT checkpoint in mpt-7b dir and save the converted checkpoint to mpt-ft-7b dir
+python mpt_ckpt_to_ft.py -i mpt-7b -o mpt-ft-7b --infer_gpu_num 1
+```
+
+## Convert Directly
```

Comment:
Very small nitpick, but could we move up this `Convert Directly` section first, before the `Pre-Download` section? I read that block first then spent 5min trying to understand why we needed to git clone a HF model, and then I read down and realized "oh I can just give it the HF Hub path, that's what I want to do"

Reply:
fixed.